vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/IssuerAgent.kt

```diff
@@ -252,14 +250,7 @@ class IssuerAgent(
         val vcId = "urn:uuid:${uuid4()}"
         val expirationDate = credential.expiration
         val timePeriod = timePeriodProvider.getTimePeriodFor(issuanceDate)
-        val subjectId = subjectPublicKey.toJsonWebKey().keyId ?: return Issuer.IssuedCredentialResult(
-            failed = listOf(
-                Issuer.FailedAttribute(
-                    scheme.vcType,
-                    DataSourceProblem("subjectPublicKey transformation error")
-                )
-            )
-        ).also { Napier.w("subjectPublicKey could not be transformed to a JWK") }
+        val subjectId = subjectPublicKey.didEncoded
```

Comment:
so here we still want the DID encoding because?

Reply:
Because we'll somehow need to encode the subject of the credential, and make a link to the holder's key.